docs/reference/overview.md

```diff
@@ -31,6 +38,12 @@ Setting this is optional. If you do not set this, the `COMPOSE_PROJECT_NAME` def
 
 Specify the file containing the compose configuration. If not provided, Compose looks for a file named  `docker-compose.yml` in the current directory and then each parent directory in succession until a file by that name is found.
 
+### COMPOSE\_API\_VERSION
+
+Specify the api version the client should report. If you get an error about
+`client and server don't have same version` you may use this environment
+variable to have the client report a version compatible with the server.
```

Comment:
Is it the specific version or a "matching" version? I think from your rewrite (noice!) same is true. So lets say matching.  

"report. typically" -> "report, typically"

Then I got to your note and hit a wall. Why are we suggesting they do this if we don't support it?  Which features do I lose if I **do** do it.  Why wouldn't I just fix the problem with the client/API mismatch ---> if this was in the discussion...let me know.  We kind of need to summarize the answers here to make the doc useful.